epuck_ros2_cpp/src/driver.cpp

```diff
@@ -98,6 +101,10 @@ class EPuckPublisher : public rclcpp::Node {
       mI2cMain = std::make_unique<I2CWrapperTest>("/dev/i2c-4");
     else
       mI2cMain = std::make_unique<I2CWrapperHW>("/dev/i2c-4");
+    mTofInitialized = tofInit(4, 0x29, 1);
+    if (!mTofInitialized) {
+      RCLCPP_WARN(get_logger(), "ToF device is not accessible!");
+    }
```

Comment:
```suggestion
    if (!mTofInitialized)
      RCLCPP_WARN(get_logger(), "ToF device is not accessible!");
```

https://github.com/cyberbotics/webots/wiki/CPP-Coding-Style#cs360-single-if-while-or-for-statement-must-be-written-without-brackets-the-statement-part-should-be-put-on-a-separate-line-in-case-of-nested-block-keep-brackets

Reply:
Not again :/ `uncrustify` is the only tool that catches those mistakes I haven't run it